packages/fcl-core/src/current-user/exec-service/wc-check.ts

```diff
@@ -0,0 +1,20 @@
+import * as logger from "@onflow/util-logger"
+import {getServiceRegistry} from "./plugins"
+
+const FC_WC_SERVICE_METHOD = "EXT/WC"
+
+// Utility to notify the user if the Walletconnect service plugin has not been loaded
+export function checkWalletConnectEnabled() {
```

Comment:
Should you add a check to make sure this doesn't run on the server side (e.g. when used in a next app)?

Reply:
Shouldn't be an issue.  Node.js is not allowed to use `currentUser` at all.